linkerd/main/src/main/scala/io/buoyant/linkerd/Main.scala

```diff
@@ -1,11 +1,13 @@
 package io.buoyant.linkerd
 
+import collection.JavaConversions._
```

Comment:
yes .. I can look at focusing on just the imports needed as opposed to blinding importing everything.

NetworkInterface.getNetworkInterfaces()  returns an Enumeration<NetworkInterface> and interface.getInetAddresses() returns an Enumeration<InetAddress>.  I get a compiler error w/o this import when attempting to loop over either Enumeration.